<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookInfo</class>
 <widget class="QWidget" name="BookInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>963</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="picture">
                 <property name="minimumSize">
                  <size>
                   <width>240</width>
                   <height>320</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>标题：</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="title">
                     <property name="text">
                      <string>标题</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>id:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="idLabel">
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_11">
                     <property name="sizeConstraint">
                      <enum>QLayout::SetDefaultConstraint</enum>
                     </property>
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>作者：</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QtCategoryList" name="autorList">
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">background-color:transparent;</string>
                       </property>
                       <property name="frameShape">
                        <enum>QFrame::NoFrame</enum>
                       </property>
                       <property name="showDropIndicator" stdset="0">
                        <bool>true</bool>
                       </property>
                       <property name="flow">
                        <enum>QListView::LeftToRight</enum>
                       </property>
                       <property name="spacing">
                        <number>10</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>描述：</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="description">
                     <property name="text">
                      <string>描述</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>分类：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QtCategoryList" name="categoriesList">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QListWidget {background-color:transparent;}
QListWidget::item {
    background-color:rgb(251, 239, 243);
    color: rgb(196, 95, 125);
	border:2px solid red;
    border-radius: 10px;
	border-color:rgb(196, 95, 125);
}
/* 鼠标在按钮上时，按钮颜色 */
 QListWidget::item:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}
</string>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="spacing">
                      <number>6</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Tags：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QtCategoryList" name="tagsList">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QListWidget {background-color:transparent;}
QListWidget::item {
    background-color:rgb(251, 239, 243);
    color: rgb(196, 95, 125);
	border:2px solid red;
    border-radius: 10px;
	border-color:rgb(196, 95, 125);
}
/* 鼠标在按钮上时，按钮颜色 */
 QListWidget::item:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}
</string>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="spacing">
                      <number>6</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>观看数：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="views">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>观看数</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="updateTick">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>80</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="starButton">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color:transparent;</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="favoriteButton">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color:transparent;</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QToolButton" name="downloadButton">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color:transparent;</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="startRead">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>开始阅读</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="styleSheet">
              <string notr="true">QTabWidget:pane {border-top:0px solid #e8f3f9;background:  transparent; }</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="epsWidget">
              <attribute name="title">
               <string>章节</string>
              </attribute>
              <layout class="QGridLayout" name="epsLayout">
               <item row="0" column="0">
                <widget class="QListWidget" name="epsListWidget">
                 <property name="styleSheet">
                  <string notr="true">QListWidget {background-color:transparent;}
QListWidget::item {
    background-color:rgb(251, 239, 243);
    color: rgb(196, 95, 125);
	border:2px solid red;
    border-radius: 10px;
	border-color:rgb(196, 95, 125);
}
/* 鼠标在按钮上时，按钮颜色 */
 QListWidget::item:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}
</string>
                 </property>
                 <property name="textElideMode">
                  <enum>Qt::ElideRight</enum>
                 </property>
                 <property name="verticalScrollMode">
                  <enum>QAbstractItemView::ScrollPerPixel</enum>
                 </property>
                 <property name="horizontalScrollMode">
                  <enum>QAbstractItemView::ScrollPerPixel</enum>
                 </property>
                 <property name="spacing">
                  <number>6</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QtCommentList" name="commentWidget">
              <attribute name="title">
               <string>评论</string>
              </attribute>
              <layout class="QGridLayout" name="commentLayout"/>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCategoryList</class>
   <extends>QListWidget</extends>
   <header location="global">.qtlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>QtCommentList</class>
   <extends>QWidget</extends>
   <header location="global">.qt_comment_list.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>startRead</sender>
   <signal>clicked()</signal>
   <receiver>BookInfo</receiver>
   <slot>StartRead()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>607</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>starButton</sender>
   <signal>clicked()</signal>
   <receiver>BookInfo</receiver>
   <slot>AddBookLike()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>favoriteButton</sender>
   <signal>clicked()</signal>
   <receiver>BookInfo</receiver>
   <slot>AddFavority()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>781</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadButton</sender>
   <signal>clicked()</signal>
   <receiver>BookInfo</receiver>
   <slot>AddDownload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddDownload()</slot>
  <slot>AddFavority()</slot>
  <slot>StartRead()</slot>
  <slot>AddBookLike()</slot>
 </slots>
</ui>
